locales/es.json

```diff
@@ -3,32 +3,32 @@
-  "Learn to code for free. freeCodeCamp's open source curriculum has helped more than 40,000 people get jobs as developers. <a class='cta-button' href='https://www.freecodecamp.org/learn'>Get started</a>": "Learn to code for free. freeCodeCamp's open source curriculum has helped more than 40,000 people get jobs as developers. <a class='cta-button' href='https://www.freecodecamp.org/learn'>Get started</a> (es)",
-  "Load More Articles": "Load More Articles (es)",
-  "footer-desc-1": "freeCodeCamp is a donor-supported tax-exempt 501(c)(3) nonprofit organization (United States Federal Tax Identification Number: 82-0779546) (es)",
-  "footer-desc-2": "Our mission: to help people learn to code for free. We accomplish this by creating thousands of videos, articles, and interactive coding lessons - all freely available to the public. We also have thousands of freeCodeCamp study groups around the world. (es)",
-  "footer-desc-3": "Donations to freeCodeCamp go toward our education initiatives, and help pay for servers, services, and staff. (es)",
-  "footer-donation": "You can <a href='https://freecodecamp.org/donate' class='inline' rel='noopener noreferrer' target='_blank'>make a tax-deductible donation here</a>. (es)",
-  "footer-trending-guides-header": "Trending Guides (es)",
+  "Learn to code — <span>free 3,000-hour curriculum</span>": "Aprende a programar — <span>Plan de Estudio Gratuito de 3,000-horas </span> (es)",
+  "If this article was helpful, <a class='cta-button' onclick='window.open(`https://twitter.com/intent/tweet?text={title}%0A%0A{url}`, `share-twitter`, `width=550, height=235` ); return false;'>tweet it.</a>": "Si este artículo te fue útil, <a class='cta-button' onclick='window.open(`https://twitter.com/intent/tweet?text={title}%0A%0A{url}`, `share-twitter`, `width=550, height=235` ); return false;'>Tuitealo</a>",
+  "If you read this far, tweet to the author to show them you care. <a class='cta-button' onclick='window.open(`https://twitter.com/intent/tweet?text=Thank%20you%20{twitter}%20for%20writing%20this%20helpful%20article%2e%0A%0A{title}%0A%0A{url}`, `share-twitter`, `width=550, height=235`); return false;'>Tweet a thanks</a>": "Si leiste hasta aquí, tweet al autor para mostrarle que te importa. <a class='cta-button' onclick='window.open(`https://twitter.com/intent/tweet?text=Thank%20you%20{twitter}%20for%20writing%20this%20helpful%20article%2e%0A%0A{title}%0A%0A{url}`, `share-twitter`, `width=550, height=235`); return false;'>Tweet un agradecimiento</a>",
```

Comment:
Do we want the tweet itself to default to English? It seems a little strange when everything else is in Spanish.

Reply:
Missed that part. I got it translated on PR #26

Reply:
fair to dismiss this PR #24